src/checklyhq-provider.ts

```diff
@@ -0,0 +1,131 @@
+/* Copyright © 2022 Seneca Project Contributors, MIT License. */
+import { ChecklyhqProviderOptions } from './types'
+import { getJSON, makeConfig, makeUrl, postJSON } from './utils'
+
+const Pkg = require('../package.json')
+
+import fetch from "node-fetch";
```

Comment:
the seneca code style mostly follows https://standardjs.com/rules.html#semicolons

Reply:
Ok